app_dart/lib/src/request_handlers/refresh_chromebot_status.dart

```diff
@@ -90,8 +90,31 @@ class RefreshChromebotStatus extends ApiRequestHandler<Body> {
           update.builderName = builder.name;
           update.luciBucket = 'luci.flutter.prod';
           await datastore.insert(<Task>[update]);
+          // Save luci task record to BigQuery only when task finishes.
+          if (update.status == Task.statusFailed || update.status == Task.statusSucceeded) {
```

Comment:
Are those two the only accepted values?

Reply:
Yes, we want to insert record only when task finishes with a fail or success.
There are status change from `New` to `In Process`, for which we do not intend to save the record (BigQuery does not support update, and it simply insert another record.